### PR TITLE
Create issue template for Housekeeping

### LINKED ISSUE #1349: Add issue type for Housekeeping
**Please add a short description of your enhancement request**

I'd like another issue type for Housekeeping.

**Describe the solution you'd like**

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or enhancement you've considered.

**Additional context**

### Kick Off:

- [ ] Ensure the enhancement is `UX refined` and aligned with UX 
_The component and/or enhancement should be published and available in the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)_
- [ ] Ensure the enhancement has been `Tech refined` with @kirbydesign/kirby-guild and this issue is updated with a clear implementation description  
_This issue should be in the [Ready to do](https://github.com/kirbydesign/designsystem/projects/1#column-4590936) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1) before starting implementation)_
- [ ] Assign yourself to this issue and move it to the [In progress](https://github.com/kirbydesign/designsystem/projects/1#column-4590937) column of the [Kirby kan-ban board](https://github.com/kirbydesign/designsystem/projects/1)

### Code:

- [ ] Create Feature Branch from [master branch](https://github.com/kirbydesign/designsystem/tree/master)
- [ ] Create a draft implementation and push to Github
- [ ] Ask a member of @kirbydesign/kirby-guild for a WIP review by creating a draft Pull Request
- [ ] Implement unit tests
- [ ] Update Cookbook Examples and Showcase, i.e. see [Radio](https://cookbook.kirby.design/home/showcase/radio)  
_Also remember to add any relevant new API documentation_

### Review:

- UX review:
  - [ ] Ensure implementation is correct in relation to the UX design and the [Kirby Styleguide on Zeplin](https://zpl.io/258pXGj)
  - [ ] With UX agree on the version of the implementation
- Code review:
  - [ ] Open a pull request (or mark the existing draft PR as `Ready for review`) and ask @kirbydesign/kirby-guild for a review  
_Remember to add `closes #issueno` to the description of the PR._
  - [ ] Once approved, merge feature branch/PR to master
- [ ] Ask a member of @kirbydesign/kirby-guild to add a link to component showcase from Kirby Component Status and update the version number

:tada: Celebrate

### PR DESCRIPTION
**Pull request by jkaltoft**
This PR closes #1349

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement (to existing content)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [x] Other... Please describe: New issue type

## What is the current behavior?

Issue Number: #1349

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/housekeeping-request.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+title: "[Housekeeping]"
+labels: NOT Prioritized, NOT Tech refined, housekeeping
+assignees: ''
+---
+
+**Short description of housekeeping request**
+
+<!-- Replace the paragraph below with your own description -->
+
+Please provide a clear and concise description of what needs housekeeping and the reason for doing it.
```
Comment (MadsBuchmann) [inline .github/ISSUE_TEMPLATE/housekeeping-request.md:13]:
```suggestion
Add a clear and concise description of what needs housekeeping along with a reason for doing it.
```

While very polite - please is not used in the paragraphs that is meant to be replaced in the other templates 😄

File: .github/ISSUE_TEMPLATE/housekeeping-request.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+**Alternatives**
+
+<!-- Replace the paragraph below with your own description -->
+
+Please provide a clear and concise description of any alternative solutions you've considered (if any).
```
Comment (MadsBuchmann) [inline .github/ISSUE_TEMPLATE/housekeeping-request.md:19]:
```suggestion
Provide a clear and concise description of any alternative solutions you've considered (if any).
```

While very polite - please is not used in the paragraphs that is meant to be replaced in the other templates 😄

File: .github/ISSUE_TEMPLATE/housekeeping-request.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+**Additional context**
+
+<!-- Replace the paragraph below with your own description -->
+
+Please add any other context about the housekeeping request e.g, links or screenshots.
```
Comment (MadsBuchmann) [inline .github/ISSUE_TEMPLATE/housekeeping-request.md:25]:
```suggestion
Add any other context about the housekeeping request e.g, links or screenshots.
```

While very polite - please is not used in the paragraphs that is meant to be replaced in the other templates 😄

**Review by MadsBuchmann — APPROVED** [review]
Minor changes - but other than that it looks good to me :D

**Review by jakobe — APPROVED** [review]
👌

**Pull request merged**